profiling/nvtx-focused-connector/kp_nvtx_focused_connector_domain.h

```diff
@@ -0,0 +1,84 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#ifndef _H_KOKKOSP_KERNEL_NVPROF_CONNECTOR_INFO
+#define _H_KOKKOSP_KERNEL_NVPROF_CONNECTOR_INFO
```

Comment:
```suggestion
#ifndef KOKKOSP_NVTX_FOCUSED_CONNECTOR_H
#define KOKKOSP_NVTX_FOCUSED_CONNECTOR_H
```

Reply:
Your suggestion looks good to me at first sight, and I assume this matches naming conventions for `#define`'ed  in other parts of Kokkos Tools/Core/Kernels. 
One question is what the `INFO` at the end of`_H_KOKKOSP_KERNEL_NVPROF_CONNECTOR_INFO` is (which is not in the new environment variable you suggest). I will look this over more and then will commit soon after I understand it myself.

Reply:
Fixed